rules/windows/powershell/powershell_accessing_win_api.yml

```diff
@@ -0,0 +1,71 @@
+title: Accessing WinAPI in PowerShell
+id: 03d83090-8cba-44a0-b02f-0b756a050306
+status: experimental
+description: Detecting use WinAPI Functions in PowerShell
+author: Nikita Nazarov
```

Comment:
Hello @NikitaStormwind!

Would you mind adding "oscd.community" to the author field?
It will help with the promotion of the initiative and engage more people in the next sprints.

Thanks!

Reply:
Done